docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -201,10 +227,24 @@ principal, you need to create Azure AD users authenticated by that principal in
 ### Assign Azure AD administrator
 
 Only the Azure AD administrator for the database can connect to it and create
-Azure AD users. Go to your database's *Active Directory admin* page
-and set the AD admin using the *Set admin* button:
+Azure AD users.
+
+<Tabs>
+<TabItem label="MySQL">
+Go to your database's **Authentication** page and set the AD
+admin using the edit button:
+
+![Set AD admin](../../../img/database-access/guides/azure/set-ad-admin.png)
+
+</TabItem>
+<TabItem label="PostgreSQL">
+Go to your database's **Authentication** page and set the AD
+admin by selecting **+ Add Azure AD Admins**:
+
+![Set AD admin](../../../img/database-access/guides/azure/set-ad-admin-postgres.png)
 
-![Set AD admin](../../../img/database-access/guides/azure/set-ad-admin@2x.png)
```

Comment:
We should keep the old steps for setting AD admin with single-server as well, can we add that back as a tab?

Reply:
Ahh, I assumed this was just a change in the Azure interface, I didn't realize it was specific to flexible servers.